pkg/scalers/aws/aws_common.go

```diff
@@ -0,0 +1,113 @@
+package aws
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/credentials/stscreds"
+	"github.com/aws/aws-sdk-go-v2/service/sts"
+
+	kedav1alpha1 "github.com/kedacore/keda/v2/apis/keda/v1alpha1"
+)
+
```

Comment:
Could you please add here general comment what is this file about? Just a few sentences

Reply:
Do you mean in this thread or in the file?
This file already existed before this PR, I've just moved from `scalers` to another package. It contains the parsing of the credentials for generating aws.Config (it's the package struct) that we use on scalers

Reply:
In the file, I know it existed before, but it is always better to put more context into the source code.

Reply:
nvm, I've just added to file xD